ansible/roles/php/tasks/main.yml

```diff
@@ -17,17 +17,21 @@
       - "php{{ php.version }}-cli"
       - "php{{ php.version }}-curl"
       - "php{{ php.version }}-dev"
-      - "php{{ php.version }}-json"
       - "php{{ php.version }}-mbstring"
       - "php{{ php.version }}-opcache"
       - "php{{ php.version }}-phpdbg"
       - "php{{ php.version }}-readline"
       - "php{{ php.version }}-xml"
       - "php{{ php.version }}-zip"
-      - php-apcu
-      - php-ast
+      - "php{{ php.version }}-apcu"
+      - "php{{ php.version }}-ast"
       - php-pear
 
+- name: install php json module for php != 8.0
+  apt:
+    name: "php{{ php.version }}-json"
+  when: php.version|string != '8.0'
```

Comment:
this solution will work until 8.1 gets released, is there no way  `< 8` ?

Reply:
`when: php.version < 8` for example

Reply:
https://github.com/scuben/vagrant-php-ubuntu/pull/1

Reply:
klappt mit 7.4, klappt es auch mit 8?

Reply:
tested now for 7.4 and 8.0. the version can be defined as string or float. only catch is that 8 (int or string) wont work (it needs to be 8.0). I guess same has applied for 7 vs. 7.4 back in the days. branch is therefore ready for merge.